docs/contributing/how-to-contribute.md

```diff
@@ -101,7 +101,7 @@ In order to accept your pull request, we need you to submit a CLA. You only need
 
 ### Development Workflow
 
-After cloning React, run `npm install` to fetch its dependencies.  
```

Comment:
Was unaware of it and it happened because of my sublime text editors configuration. It removes trailing white space. Will take care of it in future.